app/views/tours/index.html.erb

```diff
@@ -4,8 +4,8 @@
     <div class='tour-list'>
       <%= render partial: 'shared/voting_arrows', object: tour %>
       <div class='tour-box'>
-        <%= link_to tour.title, tour_path(tour) %><br>
-        Creator: <%= link_to tour.user.display_name, user_path(tour.user) %><br>
+        <%= link_to tour.title, tour_path(tour), class: 'tour-info'%><br>
```

Comment:
Let's make sure all lists of tours (on user show page & user dashboard) have this styling